update the tidb configuration documents (#3003)
pingcap/docs: tidb-configuration-file.md

- Default value: `0`
- This configuration only takes effect when `prepared-plan-cache.enabled` is `true`. When the size of the LRU is greater than `prepared-plan-cache.capacity`, the elements in the LRU are also removed.

### `txn-total-size-limit`

- The size limit of a single transaction in TiDB.
- Default value: `104857600` (in bytes)

> - In a single transaction, the total size of key-value records cannot exceed this value. The maximum value of this parameter is `10737418240` (10 GB). Note that if you have used the binlog to serve the downstream consumer Kafka (such as the `arbiter` cluster), the value of this parameter must be no more than `1073741824` (1 GB). This is because 1 GB is the upper limit of a single message size that Kafka can process. Otherwise, an error is returned if this limit is exceeded.

Reviewer: This paragraph is duplicate with [line 299](https://github.com/pingcap/docs/pull/3003/files#diff-c8655fccf1cdae606dc717101a6cf6b3R299). @TomShawn Please move your changes to line 299.

> - In a single transaction, the total size of key-value records cannot exceed this value. The upper limit of this configuration value is `10737418240` (10 GB). Note that if you have used the binlog whose downstream consumer is Kafka (such as the `arbiter` cluster), the value of this parameter must be no more than `1073741824` (1 GB). This is because 1 GB is the upper limit of a single message size that Kafka can process. In this case, an error is returned if this limit is exceeded.

Reviewer: ```suggestion
- In a single transaction, the total size of key-value records cannot exceed this value. The maximum value of this parameter is `10737418240` (10 GB). Note that if you have used the binlog to serve the downstream consumer Kafka (such as the `arbiter` cluster), the value of this parameter must be no more than `1073741824` (1 GB). This is because 1 GB is the upper limit of a single message size that Kafka can process. Otherwise, an error is returned if this limit is exceeded.
```

### max-retry-count

> - The max number of retries of each statement in pessimistic transactions. The number of retries that exceeds this limit results in error.

Reviewer: ```suggestion
- The maximum number of retries of each statement in pessimistic transactions. If the number of retries exceeds this limit, an error occurs.
```
